doc/source/glossary.rst

```diff
@@ -71,11 +73,18 @@ Glossary
         :ref:`list command <list-command>` or see the
         :ref:`Plugin Reference <instruments>`.
 
+    Iteration
+        Iteration refers to the number of times that a workload should be run
+        successfully. It is specified as a configuration parameter, either for
+        a specific workload, a specific section, or in general.
+
+    .. _glossary_job:
     Job
         An single execution of a workload. A job is defined by an associated
-        :term:`spec`. However, multiple jobs can share the same spec; e.g. Even
-        if you only have 1 workload to run but wanted 5 iterations then 5
-        individual jobs will be generated to be ran.
+        :term:`spec`. However, multiple jobs can share the same spec;
+        E.g. Even if you only have 1 workload to run but wanted 5 iterations
+        then 5 individual jobs will be generated to be run. Workload is to
```

Comment:
"Workload to blueprint as job is to product" is an incorrect analogy (see comment on "workload" below); "spec" is probably what is meant here.